.template/addons/github/template.rb

```diff
@@ -1,3 +1,37 @@
 use_source_path __dir__
 
 directory '.github'
+
+# Split README.md file to multiple wiki pages
+original_readme = File.read 'README.md'
+
+create_file '.github/wiki/Getting-Started.md' do
+  get_content_between(original_readme, '## Getting Started', '## Testing')
+end
+
+create_file '.github/wiki/Testing.md' do
+  get_content_between(original_readme, '## Testing', '## CI/CD')
+end
+
+FileUtils.mv '.github/workflows/README.md', '.github/wiki/CI-CD.md'
+
+gsub_file 'README.md', /## Getting Started.*/m do
+  <<~README
+    ## Documentation
+
+    Please check out full documentation on the [wiki](link to github wiki goes here).
+  README
+end
+
+@template_instructions.add(
+  <<~INSTRUCTION
+    * Github wiki
+
+    1. On the repository, create an empty wiki page to init the wiki directory
+    2. Pick an account to commit the wiki (your own account or a bot account)
+    3. On the selected account, generate the Github token with `repo` scope
+    4. Set up secrets
+      - `GH_EMAIL`: an email of the account
+      - `GH_TOKEN`: the generated Github token from step 3
```

Comment:
is it duplicate with 
https://github.com/nimblehq/rails-templates/blob/4a9b56c9ec35f88c34e622af3b52ae3670b0b236/.template/addons/github/.github/workflows/README.md.tt#L67-L73

Reply:
Yes Long, it is pretty similar to the readme/wiki. The idea here is to print out some message, so we know what to do next 💡 

![Screen Shot 2565-02-02 at 12 59 15](https://user-images.githubusercontent.com/6965195/152101127-1335d7f9-5a40-4c51-8b52-bb04db76a61c.png)

Should we use different the message, maybe say that the user should check the instruction in readme / wiki? what do you think?

Reply:
I prefer explicit - showing these steps directly like this to make sure the user doesn't miss them 👍